util.js

```diff
@@ -0,0 +1,10 @@
+function checkOptions(actual, supported) {
+    const actualOptions = Object.keys(actual);
+    actualOptions.forEach(option => {
+        if (!supported.includes(option)) throw new Error(`Emitter need update to support check settings option: ${option}`);
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9334725)
```suggestion
        if (!supported.includes(option)) {throw new Error(`Emitter need update to support check settings option: ${option}`);}
```